demo/src/main/scala/demo/AladinContainer.scala

```diff
@@ -153,18 +63,34 @@ object AladinContainer {
         <.div(
           // ExploreStyles.AladinContainerBody,
           Css("react-aladin-container"),
-          (resize.width, resize.height).mapN(
-            SVGTargetsOverlay(
-              _,
-              _,
-              props.fov.get,
-              world2pix.value.reuseNever,
-              List(
-                SVGTarget.CrosshairTarget(props.coordinates, Css("science-target"), 10).some,
-                gs.map(SVGTarget.CircleTarget(_, Css("guidestar"), 3))
-              ).flatten
+          (resize.width, resize.height)
+            .mapN(
+              VisualizationOverlay(
+                _,
+                _,
+                props.fov.get,
+                currentPos.value.diff(props.coordinates).offset,
+                // screenOffset,
+                GmosGeometry.shapes
+              )
```

Comment:
It's a bit hard to read in the diff, but it seems there are three checks for `resize.height.exists(_ >= 100)` in a row. Could these be consolidated to one check and use something like a React.Fragment? It may not be worth the trouble.

Reply:
You're right, we could merge them